### PR TITLE
Add missing examples

### PR DESCRIPTION
**Pull request by meghana-pedduri**
Added missing examples in the examples -> datasources folder. Added ip range example resource. Modified agent token's value to not be sensitive and thus visible.

### CODE REVIEW

File: docs/data-sources/splunk_integration.md [hunk 1]
```diff
@@ -8,7 +8,13 @@ description: |-
 
 # traceable_splunk_integration (Data Source)
 
+## Example Usage
 
+```terraform
+data "traceable_splunk_integration" "splunk" {
+  name="aditya"
```
Comment (murari54) [inline docs/data-sources/splunk_integration.md:15]: dont use name in example

File: docs/data-sources/service_id.md [hunk 2]
```diff
@@ -8,7 +8,18 @@ description: |-
 
 # traceable_service_id (Data Source)
 
-
+## Example Usage
+
+```terraform
+data "traceable_service_id" "endpoint" {
+  service_name="nginx-automation-test"
```
Comment (murari54) [inline docs/data-sources/service_id.md:15]: use generic names

File: docs/data-sources/agent_token.md [hunk 3]
```diff
@@ -8,8 +8,23 @@ description: |-
...
+## Example Usage
 
+```terraform
+data "traceable_agent_token" "example" {
+  name = "tf-provider-token-testing"
+}
 
+output "agent_token" {
+  value = data.traceable_agent_token.example.token
+  sensitive = true
```
Comment (murari54) [inline docs/data-sources/agent_token.md:20]: is it still required as we removed the value from schema?

File: docs/data-sources/service_id.md (hunk 2 above)
Comment (meghana-pedduri) [reply docs/data-sources/service_id.md:15]: done

File: docs/data-sources/splunk_integration.md (hunk 1 above)
Comment (meghana-pedduri) [reply docs/data-sources/splunk_integration.md:15]: done

File: docs/data-sources/agent_token.md (hunk 3 above)
Comment (meghana-pedduri) [reply docs/data-sources/agent_token.md:20]: its optional, this doesnt correspond to the field in the schema, its just a part of the output block indicating the show of sensitive info or to hide it

File: docs/data-sources/endpoint_id.md [hunk 4]
```diff
@@ -8,7 +8,15 @@ description: |-
 
 # traceable_endpoint_id (Data Source)
 
-
+## Example Usage
+
+```terraform
+data "traceable_endpoint_id" "endpoint" {
+  name="POST /Unauthenticated_Modification_of_external_APIs"
+  service_name="nginx-automation-test"
```
Comment (adityaKumar9006) [inline docs/data-sources/endpoint_id.md:16]: here also use genric service name

File: docs/data-sources/endpoint_id.md [hunk 5]
```diff
@@ -8,7 +8,15 @@ description: |-
...
+  enviroment_name="fintech-1"
```
Comment (adityaKumar9006) [inline docs/data-sources/endpoint_id.md:17]: use genric env name

Comment (meghana-pedduri) [reply docs/data-sources/endpoint_id.md]: done

File: docs/data-sources/endpoint_id.md (hunk 4 above)
Comment (meghana-pedduri) [reply docs/data-sources/endpoint_id.md]: done

**Review by adityaKumar9006 — APPROVED** [review]

**Pull request merged**